mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -103,7 +105,13 @@ impl Bridge<WakingMemoryStore> {
                 ))
                 .with_rules(settings.subscriptions());
             })
-            .with_store(|| PublicationStore::new_memory(BATCH_SIZE))
+            .with_store(|| {
+                // TODO: can use values from settings after storage settings is merged.
+                PublicationStore::new_memory(
```

Comment:
yeah, let's wait till settings PR is in...

Reply:
Good idea, will rebase this one after the other goes in

Reply:
I have connected settings now, however, it kinda forced me to add in some more code elsewhere as the default storage is ring_buffer and that meant added support for it in pubstore/bridge/pumps